examples/golang/envoy.yaml

```diff
@@ -0,0 +1,59 @@
+# envoy demo with golang extension enabled
+static_resources:
+  listeners:
+  - name: listener_0
+    address:
+      socket_address:
+        address: 0.0.0.0
+        port_value: 8080
+    filter_chains:
+    - filters:
+      - name: envoy.filters.network.http_connection_manager
+        typed_config:
+          "@type": type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
+          stat_prefix: ingress_http
+          http_protocol_options:
```

Comment:
are these options necessary ?

Reply:
It is not necessary in our example, but it will make it more robust - such as accept `http1.0`.

Reply:
if its not necessary then remove - its not what is being demonstrated here

Reply:
Done.